Package.resolved

```diff
@@ -10,6 +10,15 @@
           "version": "0.17.0"
         }
       },
+      {
+        "package": "CryptoSwift",
+        "repositoryURL": "https://github.com/krzyzanowskim/CryptoSwift.git",
+        "state": {
+          "branch": null,
+          "revision": "3a2acbb32ab68215ee1596ee6004da8e90c3721b",
+          "version": "1.0.0"
+        }
+      },
```

Comment:
This was auto-generated when I ran one of the 3 commands in the [Contributing guide's Tests section](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tests)